lib/manifest.js

```diff
@@ -20,16 +20,17 @@ module.exports = class Manifest {
 			this.keyTransform = key || (filepath => filepath);
 		}
 
+		this.webRoot = resolvePath(webRoot || "./", referenceDir,
+				{ enforceRelative: true });
```

Comment:
I'd rather not have to worry about distinguishing between the instance property and the local variable below, thus:

```javascript
webRoot = resolvePath(webRoot || "./", referenceDir, { enforceRelative: true });
this.webRoot = webRoot;
```

Also, for consistency I'd move this above `_#index` initialization. (I'm happy to take care of this once BAK.)

Reply:
You're right, I'd overlooked that. I still have a vague feeling that this could be simplified/clarified, but since that's just a feeling for now and it's just a minor, local detail anyway, let's not worry about it for now.